xla/backends/profiler/gpu/cupti_buffer_events.h

```diff
@@ -56,7 +56,7 @@ struct MemcpyDetails {
   int8_t dst_mem_kind;
 
   // ID of the hardware channel on which this operation ran.
-  uint32_t channel_id = -1;
+  uint32_t channel_id = static_cast<uint32_t>(-1);
```

Comment:
This resulted in an implicit type narrowing error (I believe it was C2397). The explicit static cast fixes it.